### PR TITLE
Add Counterspell Atlanta

### PR DESCRIPTION
**Pull request by aboutdavid**

### CODE REVIEW

File: src/components/Locations.js [hunk 1]
```diff
@@ -13,7 +13,8 @@ export default function Locations() {
     "Austin, TX",
     "San Francisco, CA",
     "Burlington, VT",
-    "Oshkosh, WI"
+    "Oshkosh, WI",
+    "Atlanta, GA"
```
Comment (samdev-7) [inline src/components/Locations.js:17]:
```suggestion
    "Oshkosh, WI"
```
Hold off on adding stuff here for now. We'll add it after registrations for organizers close.

File: src/index.js [hunk 2]
```diff
@@ -24,6 +25,7 @@ root.render(
           E.g. Ottawa, ON -> `/ottawa` and `Ottawa`.
         */}
         {/* <Route path="/example-city" element={<ExampleCity />} /> */}
+        {<Route path="/atlanta" element={<Atlanta />} />}
```
Comment (samdev-7) [inline src/index.js:28]:
```suggestion
        <Route path="/atlanta" element={<Atlanta />} />
```

**Review by samdev-7 — CHANGES_REQUESTED** [review]

**Comment by aboutdavid** [comment]
squashed

File: src/index.js [hunk 3]
```diff
@@ -5,6 +5,7 @@ import { BrowserRouter, Routes, Route } from "react-router-dom";
 
 import App from "./pages/App";
 import NotFound from "./pages/NotFound";
+import Atlanta from "./pages/Atlanta";
```
Comment (samdev-7) [inline src/index.js:8]: Can you move this line after the import ExampleCity comment?

**Review by samdev-7 — CHANGES_REQUESTED** [review]

**Review by samdev-7 — APPROVED** [review]

**Pull request merged**